.github/workflows/smoke-windows-cygwin.yml

```diff
@@ -27,23 +27,17 @@ jobs:
         run: |
           dir c:\tools\cygwin
           path
+          echo "c:\tools\cygwin\bin;c:\tools\cygwin\usr\bin" ^| Out-File -FilePath $env:GITHUB_PATH -Encoding utf8
```

Comment:
Does this need actions/checkout@v2 as well to work?

Reply:
going to give it a try

Reply:
this works, I can see failures from `t/02-xsdynamic.t` which is failing under cygwin & strawberry... 
I skipped it using 5688f4c822a715072f3024531e4e387519392cd2 as this seems similar to `t/03-xsstatic.t`